vendor/assets/stylesheets/dvl/components/dropdowns.scss

```diff
@@ -115,6 +115,7 @@
       &:focus {
         text-decoration: none;
         background-color: rgba($linkColor,0.04);
+        outline: 1px dotted $darkerGray;
```

Comment:
This looks really bad IMO. A dotted outline makes sense for focus, but definitely not for hover.